More Random Edition/AssetEditor.cs

```diff
@@ -59,7 +60,8 @@ public bool CanEdit<T>(IAssetInfo asset)
 			if (asset.AssetNameEquals("Data/weapons")) { return Globals.Config.Weapons.Randomize; }
 			if (asset.AssetNameEquals("Data/Boots")) { return Globals.Config.Boots.Randomize; }
 			if (asset.AssetNameEquals("Data/Monsters")) { return Globals.Config.Monsters.Randomize; }
-			if (asset.AssetNameEquals("Data/NPCDispositions")) { return Globals.Config.RandomizeNPCBirthdays; }
+			if (asset.AssetNameEquals("Data/NPCDispositions")) { return Globals.Config.NPCs.RandomizeBirthdays; }
+			if (asset.AssetNameEquals("Data/NPCGiftTastes")) { return Globals.Config.NPCs.RandomizePreferences; }
```

Comment:
Question here:
Is the RandomizePreferences setting supposed to also be for the RandomizeUniversalReferences? Currently, you can't only randomize universal preferences.

If you intended both to be separate settings, then you'll probably want to update this to check the universal setting too (includes the spoiler log, etc.).

Reply:
That's a use case I totally hadn't considered - I made them independent of each other and also cleaned up the Randomize() function.